### ISSUE TITLE
Fix S1121 FN: Support unsigned right-shift operator (>>>)

### ISSUE THREAD

**Issue by gregory-paidis-sonarsource** [SonarSource/sonar-dotnet#6308]
### Description

FN for S1121 (AssignmentInsideSubExpression).

Using C#11 unsigned right-shift operator (>>>), the rule is not raising an issue.

### Repro steps

```csharp
void foo(int a) { }

int i = 0;
foo(i >>>= 1); // FN
```

### Expected behavior

The rule should raise an issue.

### Actual behavior

The rule does not raise an issue.
